add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -186,6 +186,48 @@ const kMessageHandlers = {
   },
 
   /**
+   * Creates a layout for the remote cursor on the browser chrome,
+   * and positions it on the received coordinates.
+   *
+   * @param {Object}  message Message meant for the handler function, containing
+   *                          the following parameters in its 'data' property:
+   *                          {
+   *                            ratioX: cursor's X position (between 0-1)
+   *                            ratioY: cursor's Y position (between 0-1)
+   *                          }
+   *
+   * @param {Function} reply  Callback function, invoked with the result of the
+   *                          message handler. The result will be sent back to
+   *                          the senders' channel.
+   */
+  AddRemoteCursorOverlay: function(message, reply) {
+    let win = Services.wm.getMostRecentWindow("navigator:browser");
```

Comment:
I believe browser sharing is tied to the window that triggered the sharing (so dragging a shared tab out doesn't quite work right now). This means the current/most-recent window won't always be the right window. The sharing is triggered from bootstrap.js, but I wonder if it makes sense to stash the currently shared tab/window as part of the singleton `MozLoopAPI`...?